From 8c0d9d25d83710a4752c57cfb4a73a796aa4a2c7 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Mon, 1 Jul 2024 15:52:17 -0700
Subject: [PATCH] MAINTAINERS/MEMORY MANAGEMENT: add mm document files as
 managed files

'admin-guide/mm/' and 'mm/' directories under 'Documentation/' should be
a part of memory management subsystem, but MAINTAINERS file doesn't
mark the directories so.  Add a couple of 'F:' entry for the directories
to 'MEMORY MANAGEMENT' section.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 MAINTAINERS | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index eb1f0d3d26bf..73c13f20d927 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -14451,6 +14451,8 @@ S:	Maintained
 W:	http://www.linux-mm.org
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/akpm/mm
 T:	quilt git://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new
+F:	Documentation/admin-guide/mm/
+F:	Documentation/mm/
 F:	include/linux/gfp.h
 F:	include/linux/gfp_types.h
 F:	include/linux/memfd.h
-- 
2.39.2

